### PR TITLE
Enigma service

### PR DESCRIPTION
**Pull request by jcasanovas**

### CODE REVIEW

File: lib/enigma/cipher.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    end
+
+    # Encrypts data
+    #
+    # @param [Object] Data
+    #
+    # @return [String] Encrypted data
+    def encrypt(data)
+      cipher.key = @key
+      cipher.iv  = @iv
```
Comment (davidruizrodri) [inline lib/enigma/cipher.rb]: Use `iv` instead of `@iv`

File: lib/enigma/cipher.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
+module Enigma
+  class Cipher
+    attr_reader :iv
+
+    def initialize(iv = nil)
+      @key = enigma_key
```
Comment (davidruizrodri) [inline lib/enigma/cipher.rb]: Why didn't you add `key` as `attr_reader` like `iv`?

**Comment by davidruizrodri** [comment]
@opsidao do you think adding Hound in this project would be a good idea? :smile:

File: config.ru [hunk 3]
```diff
@@ -0,0 +1,8 @@
+require 'rubygems'
+require 'bundler'
+
+Bundler.require
```
Comment (weapp) [inline config.ru]: Something like this would be good :)
```ruby
Bundler.require(:default, Rails.env)
```

**Comment by opsidao** [comment]
@davidruizrodri sure! 

We should just add the configuration for rubocop and I can enable it in 0.2 seconds (if it doesn't explode, of course xPPP)

**Comment by opsidao** [comment]
in fact: https://github.com/wuakitv/enigma/pull/2 ;)

File: script/cibuild [hunk 4]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+## rbenv config for jenkins
+export PATH=/home/jenkins/.rbenv/bin:$PATH
+eval "$(rbenv init -)"
+
+export RAILS_ENV=test
```
Comment (davidruizrodri) [inline script/cibuild]: Shouldn't it be `RACK_ENV`?

**Comment by davidruizrodri** [comment]
+1

**Comment by weapp** [comment]
+1

**Comment by alfonsojimenez** [comment]
+1

masuno

**Pull request merged**